loader/include.go

```diff
@@ -104,17 +104,27 @@ func ApplyInclude(ctx context.Context, configDetails types.ConfigDetails, model
 			if s, err := os.Stat(f); err == nil && !s.IsDir() {
 				r.EnvFile = types.StringList{f}
 			}
+		} else {
+			envFile := []string{}
+			for _, f := range r.EnvFile {
+				f := filepath.Join(configDetails.WorkingDir, f)
```

Comment:
need to guard this with `filepath.IsAbs(r.Envfile)` as env_file can also be set to an absolute path, or relative to user's home `~`